Dockerfile

```diff
@@ -1,18 +1,26 @@
-# Take advantage of a multi-stage build to avoid dev-dependencies in the final image
-FROM node:16-alpine AS builder
+FROM node:16-alpine AS build
+ENV NODE_ENV=development
 
 WORKDIR /app
 
 COPY . .
 
-RUN yarn
+RUN yarn install
 RUN yarn build
 
-FROM node:16-alpine
+FROM node:16-slim AS dist
+ENV NODE_ENV=production
 
-WORKDIR /app
+WORKDIR /usr/local/share/graphql-inspector
+
+COPY --from=build /app/packages ./packages
+COPY --from=build /app/package.json ./
 
-COPY --from=builder /app/packages /usr/local/share/graphql-inspector
+RUN yarn install \
+    && yarn cache clean
 
-RUN ln -s /usr/local/share/graphql-inspector/cli/dist/index.js /usr/local/bin/graphql-inspector \
+RUN ln -s /usr/local/share/graphql-inspector/packages/cli/dist/index.js /usr/local/bin/graphql-inspector \
  && chmod +x /usr/local/bin/graphql-inspector
+
+RUN mkdir /app
+WORKDIR /app
```

Comment:
This should move back to where it was, as it changes seldom and thus reduces image churn when put in an earlier layer